lib/api_repo.js

```diff
@@ -77,7 +77,7 @@ function ApiRepo(opts) {
   this.pkgPrefix = opts.pkgPrefix;
   this.overridePlugins = opts.overridePlugins;
   this.zipUrl = opts.zipUrl;
-  this.isGoogleApi = !!opts.isGoogleApi;
+  this.isGoogleApi = !!opts.buildCommonProtos;
```

Comment:
I agree to both of your notes: `zipUrl` can be removed, and `buildCommonProtos` doesn't need an ApiRepo-level attribute to tell it that the protos are in a "google" directory. This will require changing the way testing works, though, since most of the testdata actually comes from a zip file currently.

I'm not currently planning to follow this change up because I want to focus on Python issues, but I think both of these are good cleanup to do in the future.